locale/sk/wivrn-dashboard.po

```diff
@@ -0,0 +1,925 @@
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the WiVRn-dashboard package.
+#
+# SPDX-FileCopyrightText: 2025 not <github.superglue094@slmail.me>
+msgid ""
+msgstr ""
+"Project-Id-Version: WiVRn-dashboard\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-07-25 22:10+0200\n"
+"PO-Revision-Date: 2025-08-02 18:48+0200\n"
+"Last-Translator: not <github.superglue094@slmail.me>\n"
+"Language-Team: English <kde-i18n-doc@kde.org>\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
In order to properly handle Slovak plural forms, this line should be updated.

For instance https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/po/sk.po?ref_type=heads#L23 lists
```
"Plural-Forms: nplurals=3; plural=(n==1) ? 1 : (n>=2 && n<=4) ? 2 : 0;\n"
```